deploy/helm/postee/values.yaml

```diff
@@ -41,11 +48,16 @@ securityContext: {}
 
 service:
   type: ClusterIP
-  portSsl: 8443
+  portSsl: 8445
   targetPortSsl: 8445
-  port: 8080
+  port: 8082
   targetPort: 8082
 
+uiService:
+  type: LoadBalancer
```

Comment:
The postee webhook by default we install as ClusterIP, this endpoint don't need to be exposed if we install in the same cluster, it will be used. In the case of postee-ui we need to expose the UI to be accessed.

Reply:
It's mostly for me to understand better. (may be not issue at all): If AquaServer is running somewhere - are there any limitation when webhook is invoked. E.g. would AquaServer be able to access internal network to send a message to webhook?

Reply:
if Postee is installed in the same cluster of AquaServer, the webhook could be invoked using ClusterIP.